awx/ui/test/e2e/utils.js

```diff
@@ -0,0 +1,8 @@
+import uuid from 'uuid';
```

Comment:
where did we land with https://github.com/ansible/awx/pull/3123/files#r252775697 ?

I agree with @kialam that this is probably an unnecessary addition to this PR. I can see the value in moving uuid generation to a single place, but that might be better done in a separate PR that updates all of the files to use the new api in one step.

Reply:
fair enough

Reply:
My general guideline for whether or not to go the abstraction route is if it simplifies code into more digestible and cleaner chunks, and is reusable. This change appears to me to create more obfuscation rather than clarity because now the reader needs to track one level deeper to see what the code is specifically doing, when the original code is just as clear and concise.